internal/provisioner/helm/README.md

```diff
@@ -0,0 +1,272 @@
+
+```console
+$ kubectl get deployments.apps my-ahoy-hello-world
+NAME                  READY   UP-TO-DATE   AVAILABLE   AGE
+my-ahoy-hello-world   1/1     1            1           102s
+
+$ kubectl get deployments.apps my-ahoy-hello-world -o yaml | grep 'image:' | xargs
+- image: nginx:1.16.0
+```
+
+This means the operator should be successfully installed.
+
+The `helm` provisioner continually reconciles BundleDeployment resources. Next, let's try deleting the my-ahoy-hello-world deployment:
+
+```bash
+kubectl delete deployments.apps my-ahoy-hello-world
+```
+
+A message saying the deployment was deleted should be returned:
+
+```console
+$ kubectl delete deployments.apps my-ahoy-hello-world
+deployment.apps "my-ahoy-hello-world" deleted
+```
+
+The provisioner ensures that all resources required for the BundleDeployment to run are accounted for on-cluster.
+So if we check for the deployment again, it will be back on the cluster:
+
+```console
+$ kubectl get deployment my-ahoy-hello-world
+NAME                  READY   UP-TO-DATE   AVAILABLE   AGE
+my-ahoy-hello-world   1/1     1            1           7s
+```
+
+The values file for the chart can be applied during install.  The values.yaml file must be put into a ConfigMap
+
+```console
+kubectl create configmap my-ahoy-values --from-file=values.yaml
+```
+
+The ConfigMap name is set in the BundleDeployment spec section
+
+```yaml
+  config:
+    valuesFrom:
+      configMapName: my-ahoy-values
+```
+
+```yaml
+apiVersion: core.rukpak.io/v1alpha1
+kind: BundleDeployment
+metadata:
+  name: my-ahoy
+spec:
+  provisionerClassName: core.rukpak.io/helm
+  config:
+    valuesFrom:
+      configMapName: my-ahoy-values
+  template:
+    metadata:
+      labels:
+        app: my-ahoy
+    spec:
+      provisionerClassName: core.rukpak.io/helm
+      source:
+        http:
+          url: https://github.com/helm/examples/releases/download/hello-world-0.1.0/hello-world-0.1.0.tgz  
+        type: http
+```
+
+### Deleting the sample chart and Local Kind Cluster
+
+To clean up from the installation, simply remove the BundleDeployment from the cluster. This will remove all references
+resources including the deployment, RBAC, and the operator namespace.
+
+> Note: There's no need to manually clean up the Bundles that were generated from a BundleDeployment resource. The plain provisioner places owner references on any Bundle that's generated from an individual BundleDeployment resource.
+
+```bash
+# Delete the combo BundleDeployment
+kubectl delete bundledeployments.core.rukpak.io my-ahoy
+```
+
+A message should show that the BundleDeployment was deleted and now the cluster state is the same as it was
+prior to installing the operator.
+
+```console
+$ kubectl delete bundledeployments.core.rukpak.io my-ahoy
+bundledeployment.core.rukpak.io "my-ahoy" deleted
+```
+
+To stop and clean up the kind cluster, delete it:
+
+```bash
+# Clean up kind cluster
+make kind-cluster-cleanup
```

Comment:
I think `kind delete cluster` also makes sense, if we want to leave the host in the same exact state as before running the cluster.

Reply:
OK.  The `make run` creates the rukpak cluster.  So I'll change the line to `kind delete clusters rukpak`